packages/appsync-modelgen-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -389,12 +527,63 @@ export class AppSyncSwiftVisitor<
 
   private getInitBody(fields: CodeGenField[]): string {
     let result = fields.map(field => {
-      const fieldName = escapeKeywords(this.getFieldName(field));
-      return indent(`self.${fieldName} = ${fieldName}`);
+      const connectionHasOneOrBelongsTo: boolean = field.connectionInfo
```

Comment:
Since you have this logic in multiple places, I would refactor this logic into a method `ConnectionHasOneOrBelongsTo` and use it in all places.